actors/builtin/miner/policy.go

```diff
@@ -171,17 +213,30 @@ type BigFrac struct {
 	denominator big.Int
 }
 
+// PARAM_SPEC
+// Initial share of a slasher's reward
 var consensusFaultReporterInitialShare = BigFrac{
 	// PARAM_FINISH
 	numerator:   big.NewInt(1),
 	denominator: big.NewInt(1000),
 }
+
+// PARAM_SPEC
+// Growth rate of a slasher's reward
 var consensusFaultReporterShareGrowthRate = BigFrac{
 	// PARAM_FINISH
 	numerator:   big.NewInt(101251),
 	denominator: big.NewInt(100000),
 }
 
+// PARAM_SPEC
+// Maximum share of slasher's reward
+var consensusFaultMaxReporterShare = BigFrac{
+	// PARAM_FINISH
+	numerator:   big.NewInt(1),
+	denominator: big.NewInt(2),
+}
```

Comment:
**Clarification needed**: hey @zixuanzh, @whyrusleeping, @anorth: are these numbers final? is there an analysis showing why we are using these numbers or we need to do it?

Reply:
they are not final and we need to do more analysis